src/main.rs

```diff
@@ -299,6 +301,20 @@ fn get_includes_from_linker_script(linker_script: &str) -> Vec<&str> {
 /// Looks for "RAM : ORIGIN = $origin, LENGTH = $length"
 // FIXME this is a dumb line-by-line parser
 fn find_ram_in_linker_script(linker_script: &str) -> Option<MemoryEntry> {
+    let linker_script = {
+        // Remove any multiline-style comments (/*...*/) in the linker script before processing.
+        // Comments within one line get replaced by an empty string.
+        // Comments that span multiple lines get replaced by newlines, in order to keep the final entry
+        // line index correct, within the original linker script.
+        let re = Regex::new(r"(?s)/\*.*?\*/").unwrap();
```

Comment:
The `?` seems to be an error?
```suggestion
        let re = Regex::new(r"(?s)/\*.*\*/").unwrap();
```